cli/hzn.go

```diff
@@ -235,7 +235,8 @@ Environment Variables:
 	exServiceListCmd := exServiceCmd.Command("list", msgPrinter.Sprintf("Display the service resources from the Horizon Exchange."))
 	exService := exServiceListCmd.Arg("service", msgPrinter.Sprintf("List just this one service. Use <org>/<svc> to specify a public service in another org, or <org>/ to list all of the public services in another org.")).String()
 	exServiceListNodeIdTok := exServiceListCmd.Flag("node-id-tok", msgPrinter.Sprintf("The Horizon Exchange node ID and token to be used as credentials to query and modify the node resources if -u flag is not specified. HZN_EXCHANGE_NODE_AUTH will be used as a default for -n. If you don't prepend it with the node's org, it will automatically be prepended with the -o value.")).Short('n').PlaceHolder("ID:TOK").String()
-	exServiceLong := exServiceListCmd.Flag("long", msgPrinter.Sprintf("When listing all of the services, show the entire resource of each services, instead of just the name.")).Short('l').Bool()
+	exServiceLong := exServiceListCmd.Flag("long", msgPrinter.Sprintf("When listing all of the services, show the entire resource of each services, instead of just the name. When listing a specific service, show more details.")).Short('l').Bool()
```

Comment:
change "show the entire resource of each services," to "show the entire service definition,"

Reply:
done